src/ceph-disk/ceph_disk/main.py

```diff
@@ -665,6 +665,19 @@ def get_dev_size(dev, size='megabytes'):
         os.close(fd)
 
 
+def is_diskdevice
```

Comment:
is_device ? I don't think we deal with anything else but devices that contain bytes